frontend/src/metabase/redux/embed/embed.ts

```diff
@@ -69,6 +69,9 @@ const interactiveEmbedSlice = createSlice({
         ...action.payload,
       };
     },
+    setIsEmbeddingSdk: (state, action: PayloadAction<boolean>) => {
```

Comment:
I don't see we're using it here yet. If this is for future PR can we introduce it then?

Reply:
@WiNloSt Good eye - this is actually a leftover code. I'll delete this!